server/models/Collective.js

```diff
@@ -3184,6 +3186,20 @@ function defineModel() {
     return metrics;
   };
 
+  Collective.prototype.setPolicies = async function (policies) {
+    for (const policy of policies) {
+      if (!Object.keys(POLICIES).includes(policy)) {
+        throw new Error(`Policy ${policy} is not supported`);
+      }
+    }
+
+    return this.update({ data: { ...this.data, policies: uniq(policies) } });
+  };
+
+  Collective.prototype.hasPolicy = function (policy) {
+    return Boolean(this.data?.policies?.includes(policy));
+  };
+
```

Comment:
This is great!

Reply:
Thanks. I added the minimum amount of methods, this can change a bit in the future.